### ISSUE TITLE
qwtw build failed on Visual Studio version 16.8

### ISSUE THREAD

**Issue by LilyWangL** [ig-or/qwtw#1]
When I use Visual Studio preview version 16.8 to build qwtw, qwtw build failed with the following error:
```
CMake Error at buildsys/ccmake.cmake:90 (message):
   this Visual Studio version 1928 look like not supported yet
Call Stack (most recent call first):
  buildsys/ccmake.cmake:637 (setPlatformName)
  qwtw/c_lib/CMakeLists.txt:13 (commonStart)
```
I found this error occur due to code `set(vs2019_versions "1920" "1921" "1922" "1923" "1924" "1925" "1926")` in _qwtw/buildsys/ccmake.cmake_. 
@ig-or Could you help take a look this issue? Thanks.

**Comment by ig-or** [comment]
Oh, sorry, will fix it.

**Comment by ig-or** [comment]
Looks like working on my side.  Please have a look if/when you will have time.

**Comment by ig-or** [comment]
Fixed  in v3.1.0 ... %)
